contributing.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+# Contributing to How We Have Been Saving The Planet This Week
+
+You can help out by suggesting stories and subjects to [helpdesk@newatoms.com](mailto:helpdesk@newatoms.com). The newsletter and it's site are open source, you can contribute directly on [our GitHub repository](https://github.com/newatoms/hwhbstptw).
+
+## Making a new newsletter on GitHub
+
+1. Make a new branch or fork for this newsletter called `newsletter/` and the issue number, eg: `newsletter/26`.
+1. Make a new newsletter file in the `newsletters` folder, the file should be named `YYYY-MM-DD-` and the issue number and the extension `.md`, eg: `2015-12-12-26.md`.
```

Comment:
They can just add one article still right, they can just add what they want to contribute to that file.